include/crocoddyl/core/integrator/rk4.hxx

```diff
@@ -0,0 +1,239 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2018-2020, LAAS-CNRS, IRI: CSIC-UPC
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#include <iostream>
```

Comment:
Cannot do this because of https://github.com/loco-3d/crocoddyl/blob/bc20d4dfc8b2efd4df61a488a15b6aea57bf861d/include/crocoddyl/core/integrator/rk4.hxx#L28